frontend/src/component/MenuList/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { memo } from 'react';
+
+import menuItem from '../../menu-items';
+
+import NavGroup from './NavGroup';
+
+const MenuList = () => {
+  const navItems = menuItem.items.map((item) => <NavGroup key={item.id} item={item} />);
+
+  return <div>{navItems}</div>;
```

Comment:
**suggestion:** **praise:** why are you wrapping navItems in div? IMO they should be wrapped in a higher level
F.E.
`<div><MenuList/></div>`